cob_bringup/bundles/base_cob4.launch

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0"?>
 <launch>
-	<arg name="robot"/>
-
+	<arg name="base"/>
 	<arg name="can_device" default="can0"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="sim" default="false"/>
 
-	<arg name="driver_yaml" value="$(arg pkg_hardware_config)/robots/common/cob4_base_driver.yaml"/>
-	<arg name="controller_yaml" value="$(arg pkg_hardware_config)/robots/common/cob4_base_controller.yaml"/>
+	<arg name="driver_yaml" value="$(arg pkg_hardware_config)/modules/base/common/base_cob4_driver.yaml"/>
```

Comment:
I'd prefer to keep `cob4` prefix before `base` prefix. what's your rational to change this?